app/javascript/dashboard/routes/dashboard/settings/integrations/Webhooks/WebhookRow.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <tr>
+    <td>
+      <div class="webhook--link">{{ webhook.url }}</div>
+      <span class="webhook--subscribed-events">
+        <span class="webhook--subscribed-label">
+          {{ $t('INTEGRATION_SETTINGS.WEBHOOK.SUBSCRIBED_EVENTS') }}:
+        </span>
+        <show-more :text="subscribedEvents" limit="60" />
```

Comment:
The `limit` prop is expected as a number
```suggestion
        <show-more :text="subscribedEvents" :limit="60" />
```